src/v2/Apps/ViewingRoom/ViewingRoomApp.tsx

```diff
@@ -67,6 +72,21 @@ const ViewingRoomApp: React.FC<ViewingRoomAppProps> = ({
   return (
     <>
       <ViewingRoomMeta viewingRoom={viewingRoom} />
+      {preview && (
+        <Box position="fixed" left={0} top={58} width="100%" zIndex={1}>
```

Comment:
@dzucconi - This {top} is the height of header here, not sure if there is a way to account for any potential banners that might also be displayed.

Reply:
Curious how we want to display multiple banners, e.g. this and the black email verification banner. 🤔

Reply:
Right now the email verification banner is displayed on top. There is no clear system how to stack those at the moments. We've discussed it and figured that email verification being close-able/ actionable / one time ask is ok to take priority and cover the preview banner until the email verification is resolved or ignored.